<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureInput</class>
 <widget class="QWidget" name="ConfigureInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConfigureInput</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabPlayer1">
      <property name="accessibleName">
       <string>Player 1</string>
      </property>
      <attribute name="title">
       <string>Player 1</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabPlayer2">
      <property name="accessibleName">
       <string>Player 2</string>
      </property>
      <attribute name="title">
       <string>Player 2</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabPlayer3">
      <property name="accessibleName">
       <string>Player 3</string>
      </property>
      <attribute name="title">
       <string>Player 3</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabPlayer4">
      <property name="accessibleName">
       <string>Player 4</string>
      </property>
      <attribute name="title">
       <string>Player 4</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabPlayer5">
      <property name="accessibleName">
       <string>Player 5</string>
      </property>
      <attribute name="title">
       <string>Player 5</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabPlayer6">
      <property name="accessibleName">
       <string>Player 6</string>
      </property>
      <attribute name="title">
       <string>Player 6</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabPlayer7">
      <property name="accessibleName">
       <string>Player 7</string>
      </property>
      <attribute name="title">
       <string>Player 7</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabPlayer8">
      <property name="accessibleName">
       <string>Player 8</string>
      </property>
      <attribute name="title">
       <string>Player 8</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabAdvanced">
      <property name="accessibleName">
       <string>Advanced</string>
      </property>
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item alignment="Qt::AlignVCenter">
    <widget class="QWidget" name="consoleInputSettings" native="true">
     <layout class="QHBoxLayout" name="buttonsBottomRightHorizontalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignVCenter">
       <widget class="QGroupBox" name="handheldGroup">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Console Mode</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioDocked">
           <property name="text">
            <string>Docked</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioUndocked">
           <property name="text">
            <string>Handheld</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="vibrationGroup">
        <property name="title">
         <string>Vibration</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QPushButton" name="vibrationButton">
           <property name="minimumSize">
            <size>
             <width>68</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>68</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">min-width: 68px;</string>
           </property>
           <property name="text">
            <string>Configure</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="motionGroup">
        <property name="title">
         <string>Motion</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QPushButton" name="motionButton">
           <property name="minimumSize">
            <size>
             <width>68</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>68</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">min-width: 68px;</string>
           </property>
           <property name="text">
            <string>Configure</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignVCenter">
       <widget class="QWidget" name="connectedControllers" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>3</number>
         </property>
         <item row="1" column="2">
          <widget class="QCheckBox" name="checkboxPlayer2Connected">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Controllers</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QCheckBox" name="checkboxPlayer4Connected">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QCheckBox" name="checkboxPlayer3Connected">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QCheckBox" name="checkboxPlayer5Connected">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <widget class="QCheckBox" name="checkboxPlayer7Connected">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QCheckBox" name="checkboxPlayer6Connected">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="checkboxPlayer1Connected">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="8">
          <widget class="QCheckBox" name="checkboxPlayer8Connected">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>2</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>3</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>4</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>5</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>6</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>7</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="8">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>8</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Connected</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QPushButton" name="buttonRestoreDefaults">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>68</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>68</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">min-width: 68px;</string>
        </property>
        <property name="text">
         <string>Defaults</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QPushButton" name="buttonClearAll">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>68</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>68</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">min-width: 68px;</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
